.gitmodules

```diff
@@ -25,7 +25,16 @@
 	path = FreeRTOS/backoffAlgorithm
 	branch = main
 	url = https://github.com/FreeRTOS/backoffAlgorithm.git
-
 [submodule "FreeRTOS/FreeRTOS-Kernel"]
 	path = FreeRTOS/FreeRTOS-Kernel
 	url = https://github.com/FreeRTOS/FreeRTOS-Kernel.git
+[submodule "aws/jobs-for-aws-iot-embedded-sdk"]
+	path = aws/jobs-for-aws-iot-embedded-sdk
+	url = git@github.com:aws/Jobs-for-AWS-IoT-embedded-sdk.git
+	branch = main
+[submodule "aws/device-defender-for-aws-iot-embedded-sdk"]
+	path = aws/device-defender-for-aws-iot-embedded-sdk
+	url = git@github.com:aws/Device-Defender-for-AWS-IoT-embedded-sdk.git
+[submodule "aws/ota-for-aws-iot-embedded-sdk"]
+	path = aws/ota-for-aws-iot-embedded-sdk
+	url = git@github.com:aws/ota-for-aWS-iot-embedded-sdk.git
```

Comment:
https? is aWS in the link with the correct cases?

Reply:
Yeah, the link will still work haha. I have fixed the issue though.